user_guide_src/source/installation/upgrade_415.rst

```diff
@@ -101,3 +101,41 @@ Content Changes
 
 All Changes
 -----------
```

Comment:
Remove this heading `All Changes` and the above `Content Changes`.
They are located below.